swift/apple/FirezoneKit/Sources/FirezoneKit/Stores/Store.swift

```diff
@@ -90,6 +113,15 @@ public final class Store: ObservableObject {
         Log.error(error)
       }
     }
+
+    // Set our sign in handler
+    self.sessionNotification.signInHandler = {
+      Task {
+        do { try await WebAuthSession.signIn(store: self) } catch { Log.error(error) }
+      }
+    }
+
+    // TODO: setup appConfiguration bindings
```

Comment:
These were inlined into `init()` because Swift can sometimes get confused and think we left `init` without initializing all stored properties.